src/main/java/com/google/sps/model/TasksClient.java

```diff
@@ -38,6 +39,14 @@
    */
   List<TaskList> listTaskLists() throws IOException;
 
+  /**
+   * Get all task lists in a user's Tasks account with the specified task list titles
+   *
+   * @return List of task lists with any of the given task list titles
+   * @throws IOException if an issue occurs with the TasksService
+   */
+  List<TaskList> listTaskLists(List<String> taskListTitles) throws IOException;
```

Comment:
Nit: `Set<String>` is potentially faster (depending on the implementation), but, more importantly, it makes clearer that the order of the titles doesn't matter, it's just a set.

Reply:
Removed - from above!

Reply:
`RESOLVED`